### DISCUSSION TITLE
URL for screenshot

### DISCUSSION THREAD

**Discussion by Joukahainen123** [M-Reimer/savescreenshot#55] [Ideas]
Hi,

It would be really nice feature to have the URL of page in case in the screenshot. This feature seems to be missing from most of the Firefox screenshot tools (likely due to some technical restriction?).

**Comment by M-Reimer** [comment]
What do you mean with "URL for screenshot"? The URL to the page you created a screenshot for? My Add-on saves this to the image metadata. So with a proper image viewer or graphics software you should be able to read that.

**Reply by Joukahainen123** [reply]
Hi,
By  URL I mean that the address bar would be visible in the screenshot. Off course the information in metadata is also nice.
